content/en/blog/_posts/2017-05-00-Draft-Kubernetes-Container-Development.md

```diff
@@ -3,6 +3,8 @@ title: " Draft: Kubernetes container development made easy "
 date: 2017-05-31
 slug: draft-kubernetes-container-development
 url: /blog/2017/05/Draft-Kubernetes-Container-Development
+author: >
+  Brendan Burns (Microsoft Azure)
 ---
 _Today's post is by __Brendan Burns, Director of Engineering at Microsoft Azure and Kubernetes co-founder._  
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.